core/new-gui/yarn.lock

```diff
@@ -12751,6 +12778,13 @@ utils-merge@1.0.1:
   resolved "https://registry.npmjs.org/utils-merge/-/utils-merge-1.0.1.tgz"
   integrity sha1-n5VxD1CiZ5R7LMwSR0HBAoQn5xM=
 
+utrie@^1.0.2:
```

Comment:
what are these libraries all are used for? isn't the only library used is `html2canvas`?

Reply:
I used "yarn add html2canvas" to install the library. I guess they are included in this library?